commodore/compile.py

```diff
@@ -125,25 +125,26 @@ def _local_setup(config, cluster_id, target):
 # pylint: disable=redefined-builtin
 def compile(config, cluster_id):
     if config.local:
-        catalog_repo = _local_setup(config, cluster_id, TARGET)
+        catalog_repo = _local_setup(config, cluster_id)
     else:
         clean_working_tree(config)
-        catalog_repo = _regular_setup(config, cluster_id, TARGET)
+        catalog_repo = _regular_setup(config, cluster_id)
 
     # Compile kapitan inventory to extract component versions. Component
     # versions are assumed to be defined in the inventory key
     # 'parameters.component_versions'
     reset_reclass_cache()
-    kapitan_inventory = inventory_reclass("inventory")["nodes"][TARGET]
-    versions = kapitan_inventory["parameters"].get("component_versions", None)
+    cluster_inventory = inventory_reclass("inventory")["nodes"]["cluster"]
+    versions = cluster_inventory["parameters"].get("component_versions", None)
     if versions and not config.local:
         set_component_overrides(config, versions)
-        update_target(config, TARGET)
```

Comment:
I believe this call to `update_target` has been a noop for a while now, but I'm not completely convinced yet.